spec/system/dashboard_system_spec.rb

```diff
@@ -423,154 +305,6 @@ def create_next_product_drive_donation(donation_date:)
-
-              OpenStruct.new manufacturer_name: manufacturer_name, total_quantity_donated: quantity_donated
-            end
-          end
-
-          describe("filtering to '#{filtered_date_range_label}'" + (set_custom_dates ? " (#{custom_dates})" : "")) do
-            before do
-              org_dashboard_page
-                .visit
-                .filter_to_date_range(filtered_date_range_label, custom_dates)
-            end
-
-            # max_manufacturers_donated_in_filtered_period + 2 because the list includes the before- & after-date-range donations, too.  But for all time it will only include after.
-
-            expected_top_manufacturer_donation_links_count = if filtered_date_range_label == "All Time"
-              [max_top_manufacturer_donations_links_count, num_manufacturers_donated_in_filtered_period +
-                1].min
-            else
-              [max_top_manufacturer_donations_links_count, num_manufacturers_donated_in_filtered_period +
-                2].min
-            end
-
-            it "shows the correct total and #{expected_top_manufacturer_donation_links_count} Top Manufacturer Donation link(s)" do
-              # "Total" is filtered by the time period
-              expected_total_manufacturer_donations = @manufacturer_donations_in_filtered_date_range.map(&:total_quantity_donated).sum
-              expect(org_dashboard_page.manufacturers_total_donations).to eq expected_total_manufacturer_donations
-
-              top_manufacturer_donation_links = org_dashboard_page.top_manufacturer_donation_links
-
-              expect(org_dashboard_page.top_manufacturer_donation_links.count).to eq expected_top_manufacturer_donation_links_count
-
-              # Expect the links to be something like "In-date-range Manufacturer 1 (21)", "In-date-range Manufacturer 2 (4,321)", etc.
-              # Strip out the item counts & manufacturer names
-              top_manufacturer_donations = top_manufacturer_donation_links.map do
-                manufacturer_name, total_donated = _1.match(/(.+) \(([0-9,]+)\)/).captures
-
-                OpenStruct.new manufacturer_name: manufacturer_name, total_quantity_donated: total_donated.delete(",").to_i
-              end
-
-              # The Top Donations list is *not* filtered by the time period
-              # It can contain both the before- and after-filtered-date-range donations
-              all_manufacturer_donations = @manufacturer_donations_in_filtered_date_range + @manufacturer_donations_outside_filtered_date_range
-
-              # By design, the setup may have created more Donations during the period than are visible in the Top Donation links
-              # Make sure each Top Donation link uniquely matches a single Donation
-              expect(all_manufacturer_donations.intersection(top_manufacturer_donations)).to match_array top_manufacturer_donations
-            end
-          end
-        end
-      end
-    end
```

Comment:
Why were these removed?

Reply:
This donation section doesn't currently exist in the dashboard page. This PR is pointed into the dashboard branch which currently has failing specs. There's a note in the [commit message](https://github.com/rubyforgood/human-essentials/pull/3857/commits/3b42cf0655d226673ff5e47aa6fec8283c8a0952). This commit is available in a [separate PR](https://github.com/rubyforgood/human-essentials/pull/3840), but that one hasn't been merged yet.